Gopkg.toml

```diff
@@ -24,26 +24,28 @@
 #   go-tests = true
 #   unused-packages = true
 
+required = ["k8s.io/code-generator/cmd/client-gen"]
+
+[[constraint]]
+  name = "k8s.io/code-generator"
+  branch = "release-1.12"
+
 [[constraint]]
   name = "k8s.io/api"
-  branch = "release-1.8"
+  version = "kubernetes-1.12.0"
 
 [[constraint]]
-  name = "k8s.io/client-go"
-  version = "v5.0.0"
+  name = "k8s.io/apimachinery"
+  version = "kubernetes-1.12.0"
 
 [[constraint]]
-  name = "github.com/libopenstorage/openstorage"
-  revision = "453be74da906d1676dca8924a339c5e25e20dce7"
+  name = "k8s.io/client-go"
+  version = "kubernetes-1.12.0"
 
 [[constraint]]
-  name = "github.com/portworx/sched-ops"
+  name = "github.com/libopenstorage/openstorage"
   branch = "master"
 
 [[override]]
   name = "github.com/coreos/bbolt"
   version = "v1.3.1-coreos.6"
-
-[prune]
```

Comment:
The global prune should be there to prune unwanted files. You can add a per-project prune rule to not prune the code-generator files.

Reply:
Done